appinventor/buildserver/src/com/google/appinventor/buildserver/BuildServer.java

```diff
@@ -453,8 +458,10 @@ public Response buildAllFromZipFile(@QueryParam("uname") String userName, File i
       return Response.status(Response.Status.FORBIDDEN).type(MediaType.TEXT_PLAIN_TYPE)
         .entity("Entry point unavailable unless debugging.").build();
 
+    boolean isAab = aab != null && aab.equals(AAB_TRUE_VALUE);
```

Comment:
You could invert the check here: `AAB_TRUE_VALUE.equals(aab)` so you don't have to do the null check.

Reply:
Changed.